sdk/js/src/EventEmitter.ts

```diff
@@ -33,37 +42,37 @@ export class EventEmitter {
 
         if (isInvalidEventKey(key)) {
             throw new Error('Not a valid event to subscribe to')
-        } else if (!this.events[key]) {
+        }  
+
+        this.removeEventListener(key, (event) => callHandlers(event, this.events[key]))
```

Comment:
might wanna make sure that `subscribe` is bound to the `this` context, otherwise this line would break if someone did something like 

```
const { subscribe } = client
subscribe('event', () => {})
```

Reply:
actually nvm this is the internal method I guess you can't do that